microsite/docs/pull_request.md

```diff
@@ -219,6 +220,35 @@ The `result` on the right is the matching [PullRequestReview][pr-scala].
 
 See [the API doc](https://developer.github.com/v3/pulls/reviews/#get-a-single-review) for full reference.
 
+### Create a review
+
+You can create a review for a pull request using `createReview`; it takes as arguments:
+
+- the repository coordinates (`owner` and `name` of the repository).
+- the pull request id.
+- `commit_id` (as part of the `CreatePRReviewRequest` object): The SHA of the commit that needs a review. Defaults to the most recent commit.
+- `body` (as part of the `CreatePRReviewRequest` object): Required when using REQUEST_CHANGES or COMMENT for the event parameter. The body text of the pull request review.
+- `event` (as part of the `CreatePRReviewRequest` object): The review action you want to perform. By leaving this blank, you set the review action state to PENDING.
+- `comments` (as part of the `CreatePRReviewRequest` object): An optional list of draft review comments.
+
+```scala mdoc:compile-only
+import github4s.domain.{CreatePRReviewRequest, PRREventApprove}
+
+val createReviewData = gh.pullRequests.createReview(
+  "47deg",
+  "github4s",
+  139,
+  CreatePRReviewRequest(Some("commit_id"), "body", PRREventApprove)  
+)
+val response = createReviewData.unsafeRunSync()
+response.result match {
+  case Left(e) => println(s"Something went wrong: ${e.getMessage}")
+  case Right(r) => println(r)
+}
+```
+
```

Comment:
```suggestion

The `result` on the right is the created [PullRequestReview][pr-scala].

```

Also I think the `pr-scala` link got removed?

Reply:
I moved it at the end of the document as you say. Or am I misreading you?

Reply:
ah yes my bad it was already there.